.gitignore~

```diff
@@ -0,0 +1,15 @@
+lib-cov
```

Comment:
This is a temporary file from your editor. It shouldn't be committed to git. You can remove it with "git rm <file>". You can also globally ignore files that end in `~` if you like (see https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).